jumble/src/components/NetworkInspector.tsx

```diff
@@ -81,6 +85,11 @@ const ModelInspector: React.FC<
   // Use our animation smoothing hook
   const { updateValue, getValue, rafRef } = useAnimationSmoothing();
 
+  // Persist inspector status across sessions.
+  useEffect(() => {
+    localStorage.setItem(NETWORK_INSPECTOR_OPEN, String(isOpen));
```

Comment:
I think this solution makes sense from a usage POV, but the question for me is whether we should have _two_ settings (one for enable/disable and another for open/closed) or just one?

See the existing setting here: https://github.com/commontoolsinc/labs/blob/0f1dc097e8a8d886e602706b394aaf619e7d3792/jumble/src/components/commands.ts#L991

Reply:
That one completely hides it so there is no tip for it either, while this retains toggle state across refreshes.

I think some people just don't want to see the inspector at all which is reasonable, I on the other hand want it around but I want it to stay in expended or collapsed based on how it was last time around.